### DISCUSSION TITLE
运行报错【No matching export in "node_modules/@tarojs/runtime/dist/runtime.esm.js" for import "isHasExtractProp"】

### DISCUSSION THREAD

**Discussion by wuye1200** [NervJS/taro#15239] [Q & A] [answered]
### 相关平台
支付宝小程序

**小程序基础库: 2.0**
**使用框架: Vue 3**

### 复现步骤
代码没有改变，只是今天突然不能运行。
提示：
[ERROR] No matching export in "node_modules/@tarojs/runtime/dist/runtime.esm.js" for import "isHasExtractProp"

    node_modules/@tarojs/plugin-html/dist/runtime.js:1:9:
      1 │ import { isHasExtractProp } from '@tarojs/runtime';

### 期望结果
正常运行

### 实际结果
报错 但是别人没有问题  重新下载代码和更换pnpm cnpm 都不行

### 环境信息
```
Taro v3.6.19

BUILD_ENV: undefined

  Taro CLI 3.6.19 environment info:
    System:
      OS: macOS 11.6
      Shell: 5.8 - /bin/zsh
    Binaries:
      Node: 16.13.0 - /usr/local/bin/node
      Yarn: 1.22.15 - ~/.yarn/bin/yarn
      npm: 8.1.0 - /usr/local/bin/npm
    npmPackages:
      @tarojs/cli: 3.5.11 => 3.5.11
      @tarojs/components: 3.5.11 => 3.5.11
      @tarojs/helper: 3.5.11 => 3.5.11
      @tarojs/plugin-framework-vue3: 3.5.11 => 3.5.11
      @tarojs/plugin-html: ^3.5.11 => 3.6.19
      @tarojs/plugin-platform-alipay: 3.5.11 => 3.5.11
      @tarojs/plugin-platform-jd: 3.5.11 => 3.5.11
      @tarojs/plugin-platform-qq: 3.5.11 => 3.5.11
      @tarojs/plugin-platform-swan: 3.5.11 => 3.5.11
      @tarojs/plugin-platform-tt: 3.5.11 => 3.5.11
      @tarojs/plugin-platform-weapp: 3.5.11 => 3.5.11
      @tarojs/router: 3.5.11 => 3.5.11
      @tarojs/runtime: 3.5.11 => 3.5.11
      @tarojs/shared: 3.5.11 => 3.5.11
      @tarojs/taro: 3.5.11 => 3.5.11
      @tarojs/taro-h5: 3.5.11 => 3.5.11
      @tarojs/webpack5-runner: 3.5.11 => 3.5.11
      babel-preset-taro: 3.5.11 => 3.5.11
      eslint-config-taro: 3.5.11 => 3.5.11
```

**Comment by OctopusRoe** [comment]
今天开了个新项目，安装@tarojs/plugin-html插件时候发现了这个问题，3.6.18后的版本兼容有问题，回退至 3.6.18 版本编译通过

**Comment by wuye1200** [comment]
我自己也发现目前版本@tarojs/plugin-html 下载不到  @官方需要看看  只要是新下载都不行

**Comment by F5F5** [comment]
![image](https://github.com/NervJS/taro/assets/15978277/41b021d9-19c9-4ea7-8263-eb8ec129b24d)
我也出现了。。。突然无限报错，不知道咋拯救了

**Comment by F5F5** [comment]
> ![image](https://private-user-images.githubusercontent.com/15978277/287617852-41b021d9-19c9-4ea7-8263-eb8ec129b24d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE2ODI1NjcsIm5iZiI6MTcwMTY4MjI2NywicGF0aCI6Ii8xNTk3ODI3Ny8yODc2MTc4NTItNDFiMDIxZDktMTljOS00ZWE3LTgyNjMtZWI4ZWMxMjliMjRkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA0VDA5MzEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYzYWM1NGJiOGEwOTU2MzA5MDZlOWRhNWVmYjNjNzI5ZTRhOGNlYjE1Y2EwNjE2MzE2ZDYxNDRkMjlmMzUyMDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ipfQ_CpVp7084CnEkoV06ME_syEHkoJFNyS-tZtzt8E) 我也出现了。。。突然无限报错，不知道咋拯救了

发现是这个的问题
```
import { Painter } from "painter-taro-3";
```
我先注释掉，pnpm dev:weapp跑起来再放开注释就能正常运行，
不注释，直接运行就会无限报错。。。

**Comment by luozejian** [comment]
> 我自己也发现目前版本@tarojs/plugin-html 下载不到 @官方需要看看 只要是新下载都不行
感谢 有用

**Comment by liujiufa** [comment]
> 今天开了个新项目，安装@tarojs/plugin-html插件时候发现了这个问题，3.6.18后的版本兼容有问题，回退至 3.6.18 版本编译通过

我怎么会退到3.6.18还是报同样的问题

**Comment by cdx111** [comment]
@oasis-cloud

**Comment by kurisu994** [comment, marked as answer]
@tarojs 所有的包版本必须一致，不能混着用。
```
@tarojs/plugin-html: 3.5.11
```
